src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -23,8 +23,8 @@
  */
 #include <mxnet/base.h>
 
-#include "../nn/dnnl/dnnl_ops-inl.h"
 #include "./elemwise_binary_op-inl.h"
+#include "../nn/dnnl/dnnl_copy-inl.h"
```

Comment:
Maybe remove the dots form the includes?

Reply:
Changed, so it will follow the guidelines